assembly/__tests__/roundtrip.spec.ts

```diff
@@ -0,0 +1,111 @@
+import "allocator/arena";
```

Comment:
You should remove this line.

Reply:
lol I pushed right before you hit submit.